### PR TITLE
make template more concise

### PR DESCRIPTION
**Pull request by dlhxu**
New Template preview:

## Summary :notebook:

[Jira ticket](https://pinecanada.atlassian.net/browse/CUS-XXX)

Is this a breaking change? **Yes/No**

**Replace with a summary of the change**. Please include relevant motivation and context. List any dependencies that are required for this change.

## Test Plan
state how this has been tested

### CODE REVIEW

**Review by HaydenMeloche — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -2,32 +2,22 @@
 
 Jira [CUS-XXX](https://pinecanada.atlassian.net/browse/CUS-XXX)
 
-Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+**Replace with a summary of the change**. Please also include relevant motivation and context. List any dependencies that are required for this change.
 
 ## Type of change :taco:
 
-Please delete options that are not relevant.
+Complete following and delete options that are not relevant, **then remove this line.**
```
Comment (SandraPetkovic) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: I think u can just delete this line entirely

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -2,32 +2,22 @@
...
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] This change requires a documentation update
 
-# How Has This Been Tested? :mag_right:
+## How Has This Been Tested? :mag_right:
 
 Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.
 Please also list any relevant details for your test configuration
```
Comment (SandraPetkovic) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: get rid of these too

Comment (dlhxu) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: sure

**Review by SandraPetkovic — APPROVED** [review]

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,32 +1,10 @@
-# Description :notebook:
+## Summary :notebook:
 
-Jira [CUS-XXX](https://pinecanada.atlassian.net/browse/CUS-XXX)
+[Jira ticket](https://pinecanada.atlassian.net/browse/CUS-XXX)
 
-Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+Is this a breaking change? **Yes/No**
 
-## Type of change :taco:
```
Comment (StevenMarrocco) [inline PULL_REQUEST_TEMPLATE.md:7]: wow, you removed the 🌮

**Review by StevenMarrocco — APPROVED** [review]

File: PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (dlhxu) [reply PULL_REQUEST_TEMPLATE.md:7]: 😿 i am sorry, it had to go

**Pull request merged**